app/src/main/java/org/shadowice/flocke/andotp/Dialogs/ManualEntryDialog.java

```diff
@@ -105,6 +108,17 @@ public void onItemSelected(AdapterView<?> adapterView, View view, int i, long l)
                     periodInput.setText(String.format(Locale.US, "%d", TokenCalculator.TOTP_DEFAULT_PERIOD));
                     algorithmInput.setSelection(algorithmAdapter.getPosition(TokenCalculator.HashAlgorithm.SHA1));
 
+                    digitsInput.setEnabled(false);
+                    periodInput.setEnabled(false);
+                    algorithmInput.setEnabled(false);
+                } else if (type == Entry.OTPType.PIN) {
+                    counterLayout.setVisibility(View.GONE);
```

Comment:
We should disable the `counter`, `period`, `digits` and `algorithm` settings here since they don't have any effect on the PIN. Maybe we should disable the "Advanced settings" drawer altogether for PINs.